docs/guide/includes/basic-connection.js

```diff
@@ -0,0 +1,21 @@
+
+const { MongoClient } = require('mongodb');
+const assert = require('assert');
+
+// Connection URL
+const url = 'mongodb://localhost:27017';
+
+// Database Name
+const dbName = 'myproject';
+
+// Create a new MongoClient
+const client = new MongoClient(url);
+
+// Use connect method to connect to the Server
+client.connect().then(async function() {
```

Comment:
You run the risk of an unhandled rejection error here. The approach I tend to take looks like:

```
async function main() {
   const client = ...
   await client.connect();

   ... other stuff ...
}

main().catch(console.dir);
```

Mostly because this will eventually gracefully degrade (by removing the `main` function) when top level await is supported. 



```

Reply:
Added https://jira.mongodb.org/browse/NODE-2243 to follow up.